constructs/lambda-service.ts

```diff
@@ -55,6 +83,10 @@ export class LambdaService extends Construct implements iam.IGrantable {
 	readonly api: apigwv2.CfnApi;
 	readonly stage: apigwv2.CfnStage;
 	readonly grantPrincipal: iam.IPrincipal;
+	readonly authorizer?: apigwv2.CfnAuthorizer;
+
+	/** Maps queue names to the queue handlers of this service, if any. */
+	public queues: Map<string, ServiceQueueFunction> = new Map();
```

Comment:
Out of curiosity, what's the application here?

Reply:
If you want to access the queues of one service from another service, you have to explicitly grant the other service permissions. To do that, you need access to the queues of the service.

We can argue whether it's good for one service to directly access the queue of another service, but I think it's useful to make that possible if anyone needs it.